MyChart.js

```diff
@@ -0,0 +1,154 @@
+const sites = require('../data/sites.json');
+const https = require('https');
+const html_parser = require('node-html-parser');
+
+module.exports = async function GetAvailableAppointments() {
+    console.log('LynnTech starting.');
+    const webData = await ScrapeWebsiteData();
+    console.log('LynnTech done.');
+    return {
+        ...sites.LynnTech,
+        ...webData,
+    };
+};
+
+let siteId = '13300632';
+let vt = '1089';
+let dept = '133001025';
+
+async function ScrapeWebsiteData() {
+    // We need to go through the flow and use a request verification token
+    let cookie = '';
+    const tokenPromise = new Promise(resolve => {
+        let response = '';
+        https
+            .get(
+                `https://mychartos.ochin.org/mychart/SignupAndSchedule/EmbeddedSchedule?id=${siteId}&vt=${vt}&dept=${dept}&view=plain&public=1&payor=-1,-2,-3,4655,4660,1292,4661,5369,5257,1624,4883&lang=english1089`,
+                res => {
+                    let body = '';
+                    // Collect the heeaders and cookies
+                    let responseHeaders = res.headers;
+                    cookie = responseHeaders['set-cookie'];
+                    res.on('data', chunk => {
+                        body += chunk;
+                    });
+                    res.on('end', () => {
+                        response = body;
+
+                        resolve(response);
+                    });
+                }
+            )
+            .on('error', e => {
+                console.error(
+                    'Error making mychart request for LynnTech: ' + e
+                );
+            })
+            .end();
+    });
+    const tokenResponse = await tokenPromise;
```

Comment:
FYI I believe you can just do
```
const tokenResponse = await new Promise(resolve => { ...
```
instead of breaking it up into two statements. You don't have to change this though, it's fine as-is.